salt/metalk8s/orchestrate/deploy_etcd_on_new_node.sls

```diff
@@ -0,0 +1,66 @@
+{%- set control_plane_ips = salt.saltutil.runner('mine.get', tgt='*', fun='control_plane_ips') %}
+
+{%- if pillar['bootstrap_id'] in control_plane_ips.keys() and control_plane_ips[pillar['bootstrap_id']] %}
+{%- set bootstrap_ip = control_plane_ips[pillar['bootstrap_id']][0] %}
+{%- else %}
+{%- set bootstrap_ip = 'localhost' %}
+{%- endif %}
+
+{%- if pillar['node_name'] in control_plane_ips.keys() and control_plane_ips[pillar['node_name']] %}
+{%- set node_ip = control_plane_ips[pillar['node_name']][0] %}
+{%- else %}
+{%- set node_ip = 'localhost' %}
+{%- endif %}
+
+{#- etcd endpoint of the new node. #}
+{%- set endpoint  = 'https://' ~ node_ip ~ ':2380' %}
+
+Generate etcd certificates on a new node:
+  salt.state:
+    - tgt: {{ pillar['node_name'] }}
+    - saltenv: {{ saltenv }}
+    - sls:
+      - metalk8s.kubeadm.init.certs.etcd-server
+      - metalk8s.kubeadm.init.certs.etcd-peer
+      - metalk8s.kubeadm.init.certs.etcd-healthcheck-client
+    - pillar:
+        repo:
+          online_mode: false
+          local_mode: false
+          host: {{ bootstrap_ip }}
+        registry_ip: {{ bootstrap_ip }}
+
+Refresh the list of etcd nodes:
+  salt.function:
+    - name: mine.update
```

Comment:
Didn't find the right syntax to only update the `etcd` endpoints, so I refresh the whole mine…

Reply:
Should be something like
```
Refresh the list of etcd nodes:
  salt.function:
    - name: mine.update
    - tgt: '*'
    - kwarg:
        mine_function: etcd_endpoints
```
But honnestly for me it's ok to refresh all the mine as we will likely not use mine in the futur for this

Reply:
I think I tried that (maybe I used `kwargs` instead of `kwarg` though) and I got Salt complaining about me doing dirty things that won't work with Salt Fluorine or sth like that.

Agreed, since we won't rely on the mine for that in the future, let's keep it as-is.